package.json

```diff
@@ -118,13 +118,14 @@
     "vinyl-buffer": "^1.0.0",
     "vinyl-paths": "^2.1.0",
     "vinyl-source-stream": "^1.1.0",
+    "vue-template-compiler": "2.6.12",
     "vueify": "^9.4.1",
     "watchify": "^3.7.0",
     "yargs": "^8.0.2"
   },
   "browserslist": "defaults",
   "engines": {
-    "node": ">=16.13.2",
-    "npm": ">=8.1.2"
+    "node": ">=16.15.0",
+    "npm": ">=8.5.5"
```

Comment:
Please also add some notes about this change in the description of the pull request

Reply:
Based on https://stackoverflow.com/questions/55921442/how-to-fix-referenceerror-primordials-is-not-defined-in-node-js :
If you are using npm >= 8.3.0
npm@^8.3.0 enables you to override the version of a package of your project's dependencies. To do so, you should add [an overrides section](https://pnpm.io/package_json#pnpmoverrides) in your package.json file:

{
  // Your current package.json
  "overrides": {
    "graceful-fs": "^4.2.10"
  }
}
and after some test, the solution above is the only on that solve the issue on npm install related to gulp@3